orderbook/src/api/create_order.rs

```diff
@@ -51,6 +51,13 @@ pub fn create_order_response(result: Result<AddOrderResult>) -> impl Reply {
             ),
             StatusCode::BAD_REQUEST,
         ),
+        Ok(AddOrderResult::InsufficientFee) => (
+            super::error(
+                "InsufficientFee",
+                "consult /token/<address>/fee for minimum fee required",
```

Comment:
Could we provide the actual token address here given that we surely know it at the time of displaying this message?

Reply:
We don't have access to the sell token in this method. I changed the error text to be less specific and a format that also renders nicer in the UI (where you don't really want to call an API endpoint).